umbral/bignum.py

```diff
@@ -81,6 +81,34 @@ def from_int(cls, num, curve: ec.EllipticCurve=None):
 
         return cls(bignum, curve_nid, group, order)
 
+    @classmethod
+    def hash_to_bn(cls, *crypto_items, params=None):
```

Comment:
As @KPrasch pointed out, `BigNum.hash_to_bn` is very awkward.  It's not like you'd ever have `BigNum.hash_to_kfrag`.  

What is *really* happening here that makes it belong on the `BigNum` class.  It's that we're creating a `BigNum` *from* something, right?

Maybe `BigNum.one_way_from_bytes`?  Or `BigNum.hash_from_crypto_items`?  Or something to that effect?

Reply:
See my comment on the `bytes` thing. My response is similar. I like the idea of renaming it, but there are more important things to do before we consider this.